scripts/install/git.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+git config --global diff.renameLimit 200000
```

Comment:
I am still convinced that these need to be using `--global`. Can you please test if these work with the `--system` flag, or some other less far-reaching location setting?

Reply:
@ccarterlandis : I tested this. The --system flag requires sudo privileges on a server. It is more common that a particular user will be running augur, and so in the case of our software --global is a more appropriate solution because we want to avoid requiring sudo wherever possible.